src/plugins/orangehrmLDAPAuthenticationPlugin/Dto/LDAPSetting.php

```diff
@@ -24,31 +24,38 @@
 
 class LDAPSetting
 {
-    private bool $enable;
-    private string $host;
-    private int $port;
-    private string $encryption;
-    private string $implementation;
+    private bool $enable = false;
+    private string $host = 'localhost';
+    private int $port = 389;
+    private string $encryption = 'none';
+    private string $implementation = 'OpenLDAP';
     private string $version = '3';
     private bool $optReferrals = false;
 
     private bool $bindAnonymously = true;
     private ?string $bindUserDN = null;
     private ?string $bindUserPassword = null;
 
-    private ?string $baseDN = null;
+    private ?string $baseDN = 'dc=example,dc=com';
```

Comment:
```suggestion
    private ?string $baseDN = '';
```

Reply:
@Super-Chama default value for Base Distingushed Name